js/sykmeldinger/sykmeldinger/PeriodeListe.js

```diff
@@ -0,0 +1,27 @@
+/* eslint arrow-body-style: ["error", "as-needed"] */
```

Comment:
Trenger du dette? Ser ikke ut som du bruker arrow-body-style?

Reply:
det er for å slippe eksplisitt return ;)